### ISSUE TITLE
License Choice

### ISSUE THREAD

**Issue by hwilliams10** [UBC-MDS/samplingsimulatorpy#87]
This issue to is discuss and provide the rationale for our license choice for our project.

**Comment by hwilliams10** [comment]
I don't think we need a Creative Commons license since we are only producing code (and basic plots), and not creating/reusing any images, visualizations, reports, or presentations.

**Comment by lisebraaten** [comment]
@hwilliams10 agreed on the Creative Commons license. 

How do you all feel about the MIT license? I personally feel that I am happy with sticking with a simple license that we are all familiar with and that serves the purpose of preserving copyright. I also do not feel that we require a restriction on distribution, which is why I would be happy with the MIT license - let me know what you all think.

**Comment by lisebraaten** [comment]
Also looking at our license currently, it appears to only be copyrighted to @tguo9. I believe we should update it regardless of which license we choose to include all team members as we did with the one in our R repo.

**Comment by hwilliams10** [comment]
> Also looking at our license currently, it appears to only be copyrighted to @tguo9. I believe we should update it regardless of which license we choose to include all team members as we did with the one in our R repo.

Good point! I agree

**Comment by hwilliams10** [comment]
> @hwilliams10 agreed on the Creative Commons license.
> How do you all feel about the MIT license? I personally feel that I am happy with sticking with a simple license that we are all familiar with and that serves the purpose of preserving copyright. I also do not feel that we require a restriction on distribution, which is why I would be happy with the MIT license - let me know what you all think.

After consulting https://choosealicense.com/ , I feel that I would also be happy sticking with the MIT license. The main difference between it and other licenses like GNU GPLv3 is that it doesn't restrict people from making and distributing closed source versions. I don't think that's an issue for us.

**Comment by YueJiangMDSV** [comment]
I am good with MIT license

**Comment by lisebraaten** [comment]
> > @hwilliams10 agreed on the Creative Commons license.
> > How do you all feel about the MIT license? I personally feel that I am happy with sticking with a simple license that we are all familiar with and that serves the purpose of preserving copyright. I also do not feel that we require a restriction on distribution, which is why I would be happy with the MIT license - let me know what you all think.
> 
> After consulting https://choosealicense.com/ , I feel that I would also be happy sticking with the MIT license. The main difference between it and other licenses like GNU GPLv3 is that it doesn't restrict people from making and distributing closed source versions. I don't think that's an issue for us.

Yes, just took a look at https://choosealicense.com/  and I agree that MIT seems the most suitable - if @tguo9 feels good with it, I think we should stick with it, just update to include all names.

**Comment by tguo9** [comment]
Yeah! I think MIT license is good enough! I don't think GPL is necessary in our case.
